lofty/src/file/file_type.rs

```diff
@@ -99,6 +99,33 @@ impl FileType {
 		}
 	}
 
+	/// Returns the extension for the `FileType` if it is known
+	///
+	/// # Examples
+	///
+	/// ```rust
+	/// use lofty::file::FileType;
+	///
+	/// assert_eq!(FileType::Mpeg.ext(), Some("mp3"));
+	/// ```
+	pub fn ext(&self) -> Option<&str> {
+		match self {
+			FileType::Aac => Some("aac"),
+			FileType::Aiff => Some("aiff"),
+			FileType::Ape => Some("ape"),
+			FileType::Flac => Some("flac"),
+			FileType::Mpeg => Some("mp3"),
+			FileType::Mp4 => Some("mp4"),
+			FileType::Mpc => Some("mpc"),
+			FileType::Opus => Some("opus"),
+			FileType::Vorbis => Some("ogg"),
+			FileType::Speex => Some("spx"),
+			FileType::Wav => Some("wav"),
+			FileType::WavPack => Some("wv"),
+			FileType::Custom(_) => None,
+		}
+	}
+
```

Comment:
```suggestion
```
I don't think `FileType` should have this method. While these are the most common extensions for these formats, it's possibly information loss.

For example:
* The `aiff` extension can also be `aifc` for compressed audio
* The `mp4` extension doesn't really say much about the file, `m4a`, `m4b`, and `m4v` are all common extensions (with many others being out there).